### ISSUE TITLE
Signal not in list of recent apps

### ISSUE THREAD

**Issue by MrStahlfelge** [WhisperSystems/Signal-Android#6610]
I have:
- [ X] searched open and closed issues for duplicates
- [X ] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
Despite set it otherwise, Signal is hidden in the list of recent apps in Android 6.0 since today's update.

**Comment by Ricain** [comment]
This is not an issue, it is a functionality to avoid reading a discussion in recent app or system screen capture of the app.

If you want to disable it:  Signal > Settings > Privacy > Disable Screen Security

**Comment by NinebitX** [comment]
@Ricain: If I have understood the report correctly, it looks like @MrStahlfelge disabled the Screen Security setting in the previous version, and the issue that is being reported is that the preference was reset after the app was updated. If this is the case, then a more complete bug report would have included the following:

**Steps to reproduce**

1. Install Signal version 4.3.2
2. Disable the Screen Security setting (Signal > Settings > Privacy > Disable Screen Security)
3. Update to Signal version 4.5.3
4. Open Signal
5. Go back to the Android home screen
6. Open the list of recent apps

**Actual result:** Signal is hidden in the list of recent apps.

**Expected result:** Signal is visible in the list of recent apps.

**Comment by Ricain** [comment]
@NinebitX my bad...

I am not sure, but when I updated Signal, I had the same issue, activating and deactivating the correct option in settings before killing Signal worked... Do not take my word for it! :angel:

**Comment by MrStahlfelge** [comment]
Yes. I activated and deactivated it, now it is listed again. Before, it was deactivated but Signal not shown in the list.

This is not the first time this happens after an update. I had this issue before (a year or more ago).

**Comment by NinebitX** [comment]
@MrStahlfelge: I think "Screen Security preference ignored after update" would be a more descriptive title to this issue.

**Comment by DeastinY** [comment]
If understood correctly the issue is that the app is not - at all - listed in the recent apps. Normally it would be blacked out, but still be there so it clould be selected to e.g. go back to.
A friend of mine just had the same issue with *Signal 4.6* on an *LG G5 H850* and we were able to resolve it by reinstalling the app. If needed I could maybe post the video he recorded to show the issue to further clarify things.

**tl;dr: Reinstalling the app fixed this. Make sure to backup everything !**

**Comment by 2-4601** [comment]
> If understood correctly the issue is that the app is not - at all - listed in the recent apps. Normally it would be blacked out, but still be there so it clould be selected to e.g. go back to.

If that's the case, then this is a duplicate of #3038.

**Comment by DeastinY** [comment]
Haha yeah, it totally sounds like it ! 
Not sure if I understood the initial issue wrongly, but what I described is absolutely the exact problem as described in #3038 . Will update if it happens again :)

**Comment by MrStahlfelge** [comment]
Yes this is the same issue. And like mentioned I had this issue before and wondered why it comes back sometimes.

By the way, it is fixed with the latest release.

**Comment by DeastinY** [comment]
Could you link to the release notes that state it is ? Because we got this issue on 4.6 which I believe is current :) 
Other than that I guess tthe issue can be be closed ? Thanks a bunch to @2-4601 anyways !

**Comment by MrStahlfelge** [comment]
No I can't link to it... but the issue is gone on my device since the update was installed. But you are right, that could be by incident.
